### ISSUE TITLE
Version 0.5.3+1 breaks flutter_facebook_login on android

### ISSUE THREAD

**Issue by nicolobozzato** [Pikaju/FlutterBraintree#16]
Hi
I recently started to have an issue with facebook plugin that in my app is managed by flutter_facebook_login (3.0.0). The issue is exposed here https://github.com/roughike/flutter_facebook_login/issues/240

The login just stop working in the way that it never return a result. 
I found out that the problem disappear when I go back to version 0.5.3 of FlutterBraintree.
From other issues I read it also seems to create problem on google_sign_in (4.1.1) though I didn't try by myself.

It seems an issue related to the onActivityResult of the Braintree plugin overriding somehow the onActivityResult of the Facebook plugin.

**Comment by Pikaju** [comment]
Should be fixed in version 0.5.3+2. Sorry for the inconvenience.

**Comment by chicrock** [comment]
@Pikaju Hi ~
Unfortunately, this error happened to me on IOS with this package version 0.6.0.
The flutter_braintree is broken my app when calling the social login action.
(flutter_facebook_login, google_sign_in)

If I removed flutter_braintree that works well.
```
# google_sign_in error
(
	0   CoreFoundation                      0x00007fff23e3cf0e __exceptionPreprocess + 350
	1   libobjc.A.dylib                     0x00007fff50ba89b2 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff23e3cd4c +[NSException raise:format:] + 188
	3   Runner                              0x000000010a947e15 -[GIDSignIn signInWithOptions:] + 276
	4   Runner                              0x000000010a94774a -[GIDSignIn signIn] + 64
	5   Runner                              0x000000010a957828 -[FLTGoogleSignInPlugin handleMethodCall:result:] + 2392
```

```
# flutter_facebook_login error
(
	0   CoreFoundation                      0x00007fff23e3cf0e __exceptionPreprocess + 350
	1   libobjc.A.dylib                     0x00007fff50ba89b2 objc_exception_throw + 48
	2   FBSDKCoreKit                        0x000000010316fdec +[FBSDKInternalUtility validateURLSchemes] + 396
	3   FBSDKLoginKit                       0x000000010330d57e -[FBSDKLoginManager logInParametersWithPermissions:serverConfiguration:] + 110
	4   FBSDKLoginKit                       0x000000010330e1a6 -[FBSDKLoginManager logIn] + 86
```

How can I fix it ?
